src/Members.js

```diff
@@ -1,34 +0,0 @@
-import React from "react";
-import gql from "graphql-tag";
-import { graphql } from "react-apollo";
-
-const Members = ({ members }) => {
-  return (
-    <div>
-      {members.map(member =>
-        <div key={member.email}>
-          {member.email}
```

Comment:
Interesting, we were already displaying users emails? Do we have other data for users such as their name?

Reply:
We were displaying Members before which was a seperate table in graph.cool. I just had to switch up the query. We can display name and email if you want, sure.